command/helper/helper.go

```diff
@@ -276,6 +277,21 @@ func FillPremineMap(
 	return nil
 }
 
+// MergeMaps is a helper method for merging multiple maps
+func MergeMaps(maps ...map[string]interface{}) map[string]interface{} {
+	// Create a resulting map
+	mergedMap := make(map[string]interface{})
+
+	// For each map, iterate over the keys and save the values
+	// into the resulting map
+	for _, m := range maps {
+		for key, value := range m {
+			mergedMap[key] = value
+		}
+	}
+	return mergedMap
+}
+
```

Comment:
Just a side note, I think we don't need this much granularity in our comments, since I would argue that there is no benefit in any comment on this method, the method is short and self-explanatory and that's great, but comments are just reiterating what the code does and bring unneccessary clutter.

Reply:
Agreed, I figure the method description is good enough 👍 

Fixed 👍
 
9723dd6